src/index.js

```diff
@@ -0,0 +1,81 @@
+import './style.css';
+
+// Ids
+const addList = document.getElementById('addList');
+const tasksContainer = document.getElementById('tasksContainer');
+const addInput = document.getElementById('addInput');
+const toDoSubmit = document.getElementById('toDoSubmit');
+
+let tasks = [];
+
+const setTask = () => {
+  const task = {
+    id: Date.now(),
+    description: addInput.value,
+    completed: false,
+  };
+  tasks = [...tasks, task];
+};
```

Comment:
- [ ] Your project is missing the project requirements as shown in the image below
![req](https://github.com/Feliperto/ToDo_List/assets/89055502/72cbc792-b651-4ca6-a757-283e1543c3d4)
- [ ] You are required to create an array with a list of todo tasks, see the code below for example
```javascript
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```
- [ ] After creating the tasks you are to iterate over the tasks, see the code below for guidance

```javascript
const populateTasks = (task, parentElement) => {
  const { description, completed, index: id } = task;
  const innerHTML = `
         <p>${description}</p>
         <p>${completed ? 'completed' : 'not completed'}</p>`;
  const newTodoListElement = document.createElement('li');
  Object.assign(newTodoListElement, { innerHTML, id });
  parentElement.appendChild(newTodoListElement);
};
```
- [ ] Once you have achieved the two steps described above when the page is loaded the todos created in the array of objects would be displayed on your webpage. Please i kindly suggest you re-read the project requirement and see the examples and implement accordingly 👍